src/srf/surfinter.cpp

```diff
@@ -341,8 +342,14 @@ void SSurface::IntersectAgainst(SSurface *b, SShell *agnstA, SShell *agnstB,
                 if(splane->ContainsPlaneCurve(sc)) {
                     SBezier bezier = sc->exact;
                     AddExactIntersectionCurve(&bezier, b, agnstA, agnstB, into);
+                    foundExact = true;
```

Comment:
Could we add more than one curve here?
If we find at least one then we skip PWL?
What if directly `return` from  here after adding the first one? This changes the current logic, but which is the correct one?

Reply:
I tried putting a return there and it fixes the immediate problem. Then I realized if for some reason there were more than one exact trim curve coincident with the plane we want them all, so i added the flag. This function needs to find all the intersections of two surfaces.